tests/dragonfly/seeder/script-genlib.lua

```diff
@@ -27,23 +53,24 @@ end
 -- lists
 -- store list of random blobs of default container/element sizes
 
-function LG_funcs.add_list(key)
-    local elements = dragonfly.randstr(LG_funcs.esize, LG_funcs.csize)
-    redis.apcall('LPUSH', key, unpack(elements))
+function LG_funcs.add_list(key, keys)
```

Comment:
```suggestion
function LG_funcs.add_list(key, huge_keys)
```

Reply:
it's not only the huge_keys but all the keys. To see if it's huge we need `keys[key]` which is `true` if it's a huge entry